app/models/SdilReturn.scala

```diff
@@ -88,9 +98,10 @@ object ReturnPeriod {
     val i = o + 1
```

Comment:
I know this is not your code but best to refactor these little bits every time we touch a file, that way we are continuously improving the readability of code 💪

Reply:
Reviewed code with Ramin.  o appears to be a reference to count plus or minus 1, however, there is no clear indication of what a better name would be.  Left at o at this time.